index.js

```diff
@@ -185,6 +187,26 @@ export const consumePurchase = (token) => Platform.select({
   android: async() => RNIapModule.consumeProduct(token),
 })();
 
+/**
+ * Should Add Store Payment (iOS only)
+ *   Indicates the the App Store purchase should continue from the app instead of the App Store. 
+ * @returns {null}
+ */
+export const getPromotedProduct = () => Platform.select({
```

Comment:
If you are supporting only `iOS` currently, please rename this `getPromotedProductIOS` in order to make users clear that only `iOS` is supported.

Reply:
One could argue that since it’s a NOP on Android, so it’s not only iOS that’s supported. Personally I would prefer NOT to use platform suffixes. There’s also existing precedent that a suffix is not required—see `clearProducts`, `clearTransaction`, `consumePurchase`, `finishTransaction`, etc.

Reply:
Yeah. You are right. This is quite not well organized yet. Leaving this as `NOP` would be fine. Thank you for your reasoning.